templates/jenkins-ephemeral/template.yml

```diff
@@ -0,0 +1,211 @@
+apiVersion: v1
```

Comment:
We should be using the out of the box template, and if it is insufficient, opening bugs to that end. I have opened https://bugzilla.redhat.com/show_bug.cgi?id=1503260

Please remove this template.